### DISCUSSION TITLE
Ocean spin-up in coupled simulations

### DISCUSSION THREAD

**Discussion by PeterHjortLauritzen** [NCAR/amwg_dev#467] [Discussion about coupled runs]
Almost all our coupled simulations show global SST cooling in the first ~50 years (and associated redistribution of heat in ocean) regardless of the sign of RESTOM and then (more of less) stabilizes. Example below from CESM projects meeting slide:

<img width="298" alt="Screenshot 2024-01-12 at 12 58 40 PM" src="https://github.com/NCAR/amwg_dev/assets/8782989/db948ece-93ae-4a1c-b6e8-32f866b0e73e">

Below is "slope chart" (regression) from @brianpm: 

![bcases_sst_slopechart_25years_regression](https://github.com/NCAR/amwg_dev/assets/8782989/3d3405ea-433e-47f6-8a3f-005a7863c0ae)

If all runs are going through a similar spin-up process, can we start our development runs from a ~50 year spun-up coupled simulation?

@klindsay28 suggested we do twin runs:

1. a standard initial run from the ATLAS initialization
2. a hybrid run starting from 54 at year 50 (*)

and see if we reach the same conclusion in both cases.

**Comment by PeterHjortLauritzen** [comment]
@adamrher @JulioTBacmeister @cecilehannay @gustavo-marques

I suggest we do the "twin" experiment with #75 (our latest "tuned" configuration) that is currently running. We could also choose a configuration that has already run 50+ years ...

**Comment by gustavo-marques** [comment]
@PeterHjortLauritzen, I just talked to @klindsay28 and we want to clarify what needs to be done for the "twin runs". Given that we will experiment with #75, we already have the two "a standard initial run from the ATLAS initialization": #75 and #73 
We now need to do #73b and #75b, which will be hybrid runs starting from 54 at year 50.
Does that make sense?

**Reply by adamrher** [reply]
That makes sense to me. If the #75b run branched off from year 50 of #54, freezes immediately, at least we will have saved time in coming to that conclusion.

But have we not yet resolved the ocean initialization issue in hybrid runs yet https://github.com/NCAR/amwg_dev/discussions/459?

**Comment by adamrher** [comment]
@brianpm some other runs that you might want to consider adding to your plot are the hybrid coordinate runs (47 and 55) and runs 26g and 46:

```
  dir =  (/"/glade/cheyenne/scratch/gmarques/archive/", \
           "/glade/cheyenne/scratch/gmarques/archive/", \
           "/glade/cheyenne/scratch/gmarques/archive/", \
           "/glade/derecho/scratch/hannay/archive/", \
           "/glade/scratch/gmarques/archive/", \

  fname = (/"b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.026g", \
            "b.e23_alpha16b.BLT1850.ne30_t232.046", \
            "b.e23_alpha16b.BLT1850.ne30_t232.047", \
            "b.e23_alpha16b.BLT1850.ne30_t232.054", \
            "b.e23_alpha16g.BLT1850.ne30_t232.055", \
```

In particular the hybrid coordinate runs warm in the first 25 years. I also wonder whether it would be instructive to do a scatter plot of climo RESTOM vs. the SST trend.

**Reply by adamrher** [reply]
This is pretty much the limit of my "colored time-series" approach for representing many coupled simulations (11 runs): 

![temp_tseries_SST long](https://github.com/NCAR/amwg_dev/assets/29961476/7ac8edfb-a8ba-4385-9202-30056d40cd64)

**Reply by brianpm** [reply]
I can easily add those other runs. I'll try to do that tomorrow and send an updated plot.

**Reply by brianpm** [reply]
Here is the updated plot with whichever of those runs have >25 years. In terms of the high-level finding that there's a cooling trend, nothing changes. 
![bcases_sst_slopechart_25years_regression](https://github.com/NCAR/amwg_dev/assets/5471157/0bc2ea31-c776-4e7b-871c-652f82905b12)

**Comment by swrneale** [comment]
I did want to mention that ENSO looks very different in the warm run (way too long period) and the original cold run (amplitude increasing again). But in the retuned 75 the initial look says it is back to something similar to 64. I guess some sensitivity to the significant SST changes would be expected.

**Comment by swrneale** [comment]
Also @brianpm where have you seen SST decreasing in all these runs? Is it systematic everywhere or is it regional, modal (i.e. PDO) looking?

**Reply by brianpm** [reply]
I didn't look at this, maybe others have. I just made the slope chart using global SST (removing land and ice-covered points).

**Comment by gustavo-marques** [comment]
@adamrher; yes, there is a short-term fix for ocean initialization, and I just posted instructions in https://github.com/NCAR/amwg_dev/discussions/459

I am happy to run #73b and #75b. But before doing so, I have a question - should we do `RUN_TYPE = startup` instead of `hybrid` and initialize both the ocean and sea ice from run 54 at year 50? This way, it will be easier to compare against the corresponding twin by starting at the same time (0001-01-01). However, I am not completely sure about the purpose of doing hybrid runs instead of startup runs with initial conditions from restart files. Let me know your thoughts.

**Reply by adamrher** [reply]
My understanding is that both hybrid and startup runs reset the clock to 0001-01-01. Only branch runs start the clock according to the time on the restart files. With hybrid, it's easier to make the cice inic consistent since you just need to stage the rpointer.cice file pointing to the year 50 restart in the new run directory, and it will prioritize that over any default cice inic. Same for clm, etc ... so in general, hybrid runs will only select the default inic if there is no rpointer file staged in the run directory for that component. Does that clarify things?

**Reply by cecilehannay** [reply]
You could start an hybrid run with any start date you want: 0001-01-01, 0100-01-01, or what ever you want. 

In this case, you should start with a start date of 0001-01-01.

**Comment by gustavo-marques** [comment]
Yes, thanks for clarifying. I was confused because `RUN_STARTDATE: 0043-01-01`in 064b, but I will keep the default 0001-01-01 in 73b and 75b.

**Reply by adamrher** [reply]
I wouldn't have expected that as 64b is one of the failed hybrid runs. I'll defer to @cecilehannay as maybe I don't understand this as well as I thought. But I don't think there's any harm in just setting the hybrid start date to 0001-01-01. I can always adjust the calendar in post processing if I need to line up the a time-series differently.

**Reply by cecilehannay** [reply]
You could start an hybrid run with any start date you want: 0001-01-01, 0100-01-01, or what ever you decide.

I recommend that we start 73spunup and 75spunup (73b and 75b if you prefer) as hybrid runs of 54 at year 0050-01-01 with a start date at 0001-01-01. 
What we want to test the hypothesis is that we obtain the same conclusion whether we use the ATLAS initial condition or a spinup state. If we reach the same conclusion with the two initial conditions, it means that we could use a spunup state for our development runs and we will save 50 years in every development run. All the development runs will start from year 0001-01-01. Similar to all the b1850 cesm2 runs that start as hybrid runs from the LENS at year 402. But we use a start date of 0001-01-01 and not 0402-01-01.  

What we tested in 64b,c,... was completely different. The purpose was not to test a spunup state but it was to test the robustness of the Lab Sea freeze and to see if a small perturbation at different stage of the run would send you us to another state of the Lab Sea. This is why I use a start date of 0043-01-01. 
- By starting 64b with a starting date of 0043-01-01, we could compare 64 and 64b at the same year. Say 64 froze at year 50, I could look if 64b at year 50 was frozen too. 
- While if I had started 64b from 64 at year 43 with a start date of 0001-01-01, I would have had to compare 64 at year 50 with 64b at year 7. 

The purpose of the spunup ocean is completely different and that run should start at year 0001-01-01. I hope this makes sense to you

**Reply by gustavo-marques** [reply]
Merci, @cecilehannay! 
Runs 073b and 075b have started.

**Reply by adamrher** [reply]
Here's a first look at the new runs 73b and 75b:

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/0f35c3c7-ef9b-4abe-ad39-e2c283c2196d)

**Reply by gustavo-marques** [reply]
@klindsay28 found an issue (or unintended experiment) in [#73b](https://github.com/NCAR/amwg_dev/issues/468) and [#75b](https://github.com/NCAR/amwg_dev/issues/469). In both runs, I unintentionally kept the original initial conditions in the user_nl files for ice and land. So, only the ocean and atm are reading the restart from [#054](https://github.com/NCAR/amwg_dev/issues/420) at 0055-01-01. To inform us how the initial conditions of the sea ice affect the initial SST cooling, I will run 73c and 75c with all components reading the restart from 054.

**Reply by gustavo-marques** [reply]
@adamrher, can you please update your plot? I am curious to see how the Global SST cooling evolved in 75b.

**Reply by adamrher** [reply]
updated the plot above.

**Comment by cecilehannay** [comment]
We compared diags for:   
- 75 vs 66 (yrs 50-70): [diags](https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.075/atm/index.html)
- 75c vs 66c (yrs 10-30): [diags](https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.075c/atm/b.e23_alpha16g.BLT1850.ne30_t232.075c_10_30_vs_b.e23_alpha16g.BLT1850.ne30_t232.066c_10_30/)

Comparison is complicated by the fact that 66 freezes over.
